x-pack/plugins/ml/public/application/settings/anomaly_detection_settings.tsx

```diff
@@ -0,0 +1,229 @@
+
+import { useNotifications } from '../contexts/kibana';
+import { ml } from '../services/ml_api_service';
+
+interface Props {
+  canGetFilters: boolean;
+  canCreateFilter: boolean;
+  canGetCalendars: boolean;
+  canCreateCalendar: boolean;
+}
+
+export const AnomalyDetectionSettings: FC<Props> = ({
+  canGetFilters,
+  canCreateFilter,
+  canGetCalendars,
+  canCreateCalendar,
+}) => {
+  const [calendarsCount, setCalendarsCount] = useState(0);
+  const [filterListsCount, setFilterListsCount] = useState(0);
+
+  const { toasts } = useNotifications();
+
+  useEffect(() => {
+    loadSummaryStats();
+  }, []);
+
+  async function loadSummaryStats() {
+    // Obtain the counts of calendars and filter lists.
+    if (canGetCalendars === true) {
+      try {
+        const calendars = await ml.calendars();
+        setCalendarsCount(calendars.length);
+      } catch (e) {
+        toasts.addDanger(
+          i18n.translate('xpack.ml.settings.anomalyDetection.loadingCalendarsCountErrorMessage', {
+            defaultMessage: 'An error occurred obtaining the count of calendars',
+          })
+        );
+      }
+    }
+
+    if (canGetFilters === true) {
+      try {
+        const filterLists = await ml.filters.filtersStats();
+        setFilterListsCount(filterLists.length);
+      } catch (e) {
+        toasts.addDanger(
+          i18n.translate('xpack.ml.settings.anomalyDetection.loadingFilterListCountErrorMessage', {
+            defaultMessage: 'An error occurred obtaining the count of filter lists',
+          })
+        );
+      }
+    }
+  }
+
+  return (
+    <Fragment>
+      <EuiPageContent className="mlSettingsPage__content" horizontalPosition="center">
+        <EuiPageContentHeader>
+          <EuiTitle>
+            <h2>
+              <FormattedMessage
+                id="xpack.ml.settings.anomalyDetection.anomalyDetectionTitle"
+                defaultMessage="Anomaly Detection"
+              />
+            </h2>
+          </EuiTitle>
+        </EuiPageContentHeader>
+
+        <EuiFlexGroup gutterSize="xl">
+          <EuiFlexItem grow={5}>
+            <EuiTitle size="s">
+              <h3>
+                <FormattedMessage
+                  id="xpack.ml.settings.anomalyDetection.calendarsTitle"
+                  defaultMessage="Calendars"
+                />
+              </h3>
+            </EuiTitle>
+            <EuiSpacer size="s" />
+            <EuiTextColor color="subdued">
+              <p>
+                <small>
+                  Calendars contain a list of scheduled events for which you do not want to generate
+                  anomalies, such as planned system outages or public holidays.
+                </small>
+              </p>
+            </EuiTextColor>
+            <EuiSpacer size="m" />
+            <EuiFlexGroup alignItems="center">
+              {canGetCalendars && (
+                <EuiFlexItem grow={false} style={{ display: 'block' }}>
```

Comment:
why do you override a flex item to be a regular block?

Reply:
Done in https://github.com/elastic/kibana/pull/68663/commits/e67c1fb40fd8180d877148fc18d379610b21e5d6